nexuse2e-core/src/main/java/org/nexuse2e/util/DateWithTimezoneSerializer.java

```diff
@@ -0,0 +1,22 @@
+package org.nexuse2e.util;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonPrimitive;
+import com.google.gson.JsonSerializationContext;
+import com.google.gson.JsonSerializer;
+
+import java.lang.reflect.Type;
+import java.util.Date;
+
+public class DateWithTimezoneSerializer implements JsonSerializer<Date> {
+    @Override
+    public JsonElement serialize(Date date, Type type, JsonSerializationContext jsonSerializationContext) {
+        if (date == null) {
+            return null;
+        } else {
+            String dateString = DateUtil.localTimeToTimezone(date, DateUtil.getTimezone(), "yyyy-MM-dd HH:mm:ss z");
```

Comment:
Why not use ISO8601 with timezone?

Reply:
We are only showing the date in the frontend (we don't parse it). Since the user can set their own timezone manually, we need a nice display like "2021-03-18 07:27:26 GMT".